src/store/index.ts

```diff
@@ -4,22 +4,20 @@ import Vuex from 'vuex'
 
 // Store modules
 import * as States from './state'
-import * as Getters from './getters'
-import * as Mutations from './mutations'
-import * as Actions from './actions'
+import getters from './getters'
+import mutations from './mutations'
+import actions from './actions'
 
 /**
  * Configures and returns Vuex Store.
  */
 export function getVuexStore () {
   Vue.use(Vuex)
 
-  const store = new Vuex.Store<any>({
+  return new Vuex.Store<any>({
     state: { ...States },
-    getters: { ...Getters },
-    mutations: { ...Mutations },
-    actions: { ...Actions }
+    getters,
+    mutations,
+    actions
```

Comment:
I cleaned up the getters, actions and mutators to follow convention.  No additional changes were made.